src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala

```diff
@@ -45,7 +49,7 @@ private[almaren] case class MainHTTP(
 
     import df.sparkSession.implicits._
 
-    val result = df.mapPartitions(partition => {
+    val result = df.withColumn("""`__URL__`""",urlEncodeing('__URL__)).mapPartitions(partition => {
```

Comment:
What's this ?

Reply:
Encoding the URL field with UDF which is coming from the Dataframe. Maybe I understood wrong sorry for that, if it is wrong please comment out what we want to do in the below issue

```URL
https://github.com/modakanalytics/http.almaren/issues/4
```

Reply:
@badrinathpatchikolla , this should not be default part of the code. You should add the udf and than document it.

Reply:
Sure @mantovani  Will make this as Util and document it in the readMeSteps

Reply:
Updated in the ReadMe File,Please review it